desk/ted/posthog.hoon

```diff
@@ -1,9 +1,9 @@
-::  posthog: submit an log event to PostHog
+::  posthog: submit an log event to Posthog
 ::
 /-  spider
 /+  io=strandio, l=logs
 ::
-=+  posthog-key='phc_GyI5iD7kM6RRbb1hIU0fiGmTCh4ha44hthJYJ7a89td'
+=+  posthog-key='phx_5Cee2IQzFw5XapO8xiXZj9VvQEAcJY2NRaEGly9ZKXbb9sf'
```

Comment:
I don't recognize this new key?

Reply:
@arthyn any way to get this into a desk without storing in plaintext on GitHub?

Reply:
ugh I've wanted something like an env var store for a while now. we could abuse settings store maybe? Or just make this a piece of state in the logs agent that you have to poke in. The problem with any of those is that we have to find a way to trigger those pokes that doesn't reveal the key? Either way someone could trivially read the state so idk if that really helps us. I think what you really want is just an authenticated service that accepts requests.

Reply:
cool, fine to leave in for now but let's make sure the key is for the correct project

Reply:
We are now using a personal key for this that only has write access to Posthog activities. If someone abuses it, it is trivial to revoke, unlike the previous key which is probably used in many other places (and which, unfortunately, is already out!)